spec/granite/form/model/validations/nested_spec.rb

```diff
@@ -143,6 +143,106 @@
       it do
         expect { subject.valid? }.not_to raise_error
         expect(subject).not_to be_valid
+        expect(subject.errors.count).to eq(1)
+      end
+    end
+  end
+
+  context 'object field is invalid and has representation of that field' do
+    before do
+      stub_class(:validated_object) do
+        include Granite::Form::Model::Validations
+        include Granite::Form::Model::Attributes
```

Comment:
```suggestion
      stub_model(:validated_object) do
```

Reply:
Why not?

Reply:
I missed that it's `stub_class  -> stub_model`